controller/setting_controller.go

```diff
@@ -665,7 +665,7 @@ func (sc *SettingController) updateCNI() error {
 	}
 
 	if !volumesDetached {
-		return errors.Errorf("failed to apply %v setting to Longhorn workloads when there are attached volumes", types.SettingNameStorageNetwork)
+		return &types.ErrorNotNow{Reason: fmt.Sprintf("failed to apply %v setting to Longhorn workloads when there are attached volumes", types.SettingNameStorageNetwork)}
```

Comment:
This means that this error will never be printed so the users might not understand why the new setting doesn't get applied?

Reply:
First, the setting is not necessarily new.  Second, this is just making the error type distinguishable from any other error, so we can decide to handle it differently when it is returned.  But here we are just instantiating it.

Reply:
Talked to @james-munson , Looks like the function `utilruntime.HandleError(err)` still print out the error so it is good to me